### PR TITLE
Add unit test for unicode string in from address

### PR DESCRIPTION
**Pull request by mjmayer**
Adds test to validate that unicode characters work as part of the sender name.

### CODE REVIEW

File: message_test.go [hunk 1]
```diff
@@ -97,6 +97,22 @@ Content-Disposition: attachment; filename="invite.ics"
 QkVHSU46VkNBTEVOREFSClZFUlNJT046Mi4wClBST0RJRDotLy9tYWlscHJvdG8vL01haWxQcm90bwpDQUxTQ0FMRTpHUkVHT1JJQU4KQkVHSU46VkVWRU5UCkRUU1RBTVA6MjAxNzAxMTZUMTU0MDAwClVJRDpteWNvb2xldmVudEBtYWlscHJvdG8KCkRUU1RBUlQ7VFpJRD0iQW1lcmljYS9OZXdfWW9yayI6MjAxNzAxMThUMTEwMDAwCkRURU5EO1RaSUQ9IkFtZXJpY2EvTmV3X1lvcmsiOjIwMTcwMTE4VDEyMDAwMApTVU1NQVJZOlNlbmQgYW4gZW1haWwKTE9DQVRJT046VGVzdApFTkQ6VkVWRU5UCkVORDpWQ0FMRU5EQVI=
 --_=test=_bbd1e98aa6c34ef59d8d102a0e795027--`
```
Comment (ruffrey) [inline message_test.go:99]: Does this compile? Would have expected :=

Comment (mjmayer) [reply message_test.go:99]: Yes, this compiles and passes tests

**Review by ruffrey — APPROVED** [review]
